### ISSUE TITLE
Use without jsdom

### ISSUE THREAD

**Issue by joshkel** [hustcc/jest-canvas-mock#124]
Would you be open to jest-canvas-mock exposing its HTMLCanvasElement functionality directly, instead of only setting it up within the DOM globals?

Background:

I'd like to test several functions that use [Chart.js](https://www.chartjs.org/).

Chart.js itself doesn't depend on a DOM implementation and works just fine with [node-canvas](https://www.npmjs.com/package/canvas).

I'd like for my tests to run as fast as possible. [Initializing jsdom takes time](https://jestjs.io/blog/2021/05/25/jest-27#flipping-defaults), and doing all the image processing necessary to render an in-memory canvas in Node also takes time.

jest-canvas-mock has a fully-featured mock implementation of the canvas API. If I could use it to pass to Chart.js in my tests, I expect they'd run faster than jsdom or node-canvas and that their results would be easier to verify than with node-canvas.

If this change is worth considering, I can work up a PR.

**Comment by lifeinchords** [comment]
Great idea, any luck setting this up?

**Comment by joshkel** [comment]
@lifeinchords No. We're using our own in-house mock that mocks just enough to work for our purposes; it's ugly but functional.

https://gist.github.com/joshkel/5c62dade4da0aa1f8abe9bb1185f5fd1
